src/tracer/MOM_generic_tracer.F90

```diff
@@ -61,13 +61,12 @@ module MOM_generic_tracer
   use MOM_hor_index, only : hor_index_type
   use MOM_io, only : file_exists, read_data, slasher, vardesc, var_desc
   use MOM_restart, only : register_restart_field, query_initialized, MOM_restart_CS
-  use MOM_spatial_means, only : global_area_mean
```

Comment:
@jkrasting is right, this commit did undo the changes from https://github.com/NOAA-GFDL/MOM6/pull/351/files . I'm guessing this happened because he was merging in changes from a file that is being copied in via the XML and isn't even on GitHub.